storage/src/journal/fixed.rs

```diff
@@ -229,6 +229,48 @@ impl<B: Blob, E: Storage<B>, const N: usize> Journal<B, E, N> {
         Ok(item_position + self.cfg.items_per_blob * newest_blob.0)
     }
 
+    /// Rewind the journal by `num` items.
+    ///
+    /// The journal is not synced after rewinding.
+    pub async fn rewind(&mut self, num: u64) -> Result<(), Error> {
```

Comment:
I wonder if it makes sense to use an absolute index here rather than a diff?

Reply:
Seems like callers will need to do some sort of calc to get a diff (and we do one internally too)

Reply:
yes and forgetting API is absolute position based so this would make it more consistent. changed.